CefSharp/WebBrowserExtensions.cs

```diff
@@ -183,6 +186,54 @@ public static Task<string> GetTextAsync(this IWebBrowser browser)
         /// <summary>
         /// Execute some Javascript code in the context of this WebBrowser. As the method name implies, the script will be
         /// executed asynchronously, and the method therefore returns before the script has actually been executed.
+        /// This simple helper extension will encapsulate params in single quotes (unless int, uint, etc)
+        /// </summary>
+        /// <param name="browser">The ChromiumWebBrowser instance this method extends</param>
+        /// <param name="methodName">The javascript method name to execute</param>
+        /// <param name="args">the arguments to be passed as params to the method</param>
+        public static void ExecuteScriptAsync(this IWebBrowser browser, string methodName, params object[] args)
+        {
+            var stringBuilder = new StringBuilder();
+            stringBuilder.Append(methodName);
+            stringBuilder.Append("(");
+            for (int i = 0; i < args.Length; i++)
+            {
+                var obj = args[i];
+                if(obj == null)
+                {
+                    stringBuilder.Append("null");
+                }
+                else
+                {
+                    var encapsulateInSingleQuotes = !numberTypes.Contains(obj.GetType());
+                    if(encapsulateInSingleQuotes)
+                    {
+                        stringBuilder.Append("'");
+                    }
+
+                    stringBuilder.Append(args[i].ToString());
+
+                    if (encapsulateInSingleQuotes)
+                    {
+                        stringBuilder.Append("'");
+                    }
+                }
+
+                stringBuilder.Append(", ");
+            }
+            
+            //Remove the trailing comma
+            stringBuilder.Remove(stringBuilder.Length - 2, 2);
```

Comment:
for the case `args.Length == 0` this would remove the last character of the method name and the (
e.g.
`ExecuteScriptAsync (browser, "console.log");` -> `"console.lo);"`